SU-HSDO/suhumsci
# HSD8-578 Added padding for cards in ECK fields

# READY FOR REVIEW

# Summary
- Added padding to cards that are used in ECK displays

# Needed By (Date)
- ASAP

# Urgency
- High - for philit that is launching in a couple weeks.

# Steps to Test
1. Checkout this branch
1. sync philit `blt drupal:sync --site=philit --partial`
1. Edit event displays at `/admin/structure/types/manage/hs_event/display-layout/default` and add the "Sessions" field to the main content
1. view the page `/node/2216` and ensure space is between each card. 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
